core/servers/chat/mrc_multiplexer.js

```diff
@@ -0,0 +1,325 @@
+/* jslint node: true */
+'use strict';
+
+//  ENiGMA½
+const Log                   = require('../../logger.js').log;
+const { ServerModule }      = require('../../server_module.js');
+const Config                = require('../../config.js').get;
+const { Errors }            = require('../../enig_error.js');
+const SysProps              = require('../../system_property.js');
+const StatLog               = require('../../stat_log.js');
+
+//  deps
+const net                   = require('net');
+const _                     = require('lodash');
+const os                    = require('os');
+
+
+// MRC
+const protocolVersion   = '1.2.9';
+const lineDelimiter     = new RegExp('\r\n|\r|\n');
+
+const ModuleInfo = exports.moduleInfo = {
+    name        : 'MRC',
+    desc        : 'An MRC Chat Multiplexer',
+    author      : 'RiPuk',
+    packageName : 'codes.l33t.enigma.mrc.server',
+    notes       : 'https://bbswiki.bottomlessabyss.net/index.php?title=MRC_Chat_platform',
+};
+
+const connectedSockets = new Set();
+
+exports.getModule = class MrcModule extends ServerModule {
+    constructor() {
+        super();
+
+        this.log            = Log.child( { server : 'MRC' } );
+        this.config         = Config();
```

Comment:
I _think_ this is only used locally, so probably just want `const config = Config()`